### ISSUE TITLE
Static does not care about screen

### ISSUE THREAD

**Issue by superpupervlad** [qtile/qtile#1863]
# Issue description
So static function does not consider screen variable, all this functions will be working identically
```
window.static(screen=0, x=0, y=0, width=0, height=0)
window.static(screen=1, x=0, y=0, width=0, height=0)
window.cmd_static(screen=0, x=0, y=0, width=0, height=0)
window.cmd_static(screen=1, x=0, y=0, width=0, height=0)
```

# Qtile version
0.15.1-1

# Simple example
This code will place window in (10,20) on 0 screen, not on 1.
```
@hook.subscribe.client_new
def func(new_window):
	if new_window.match(wmclass="pavucontrol"):
		new_window.static(screen=1, x=10, y=20, width=100, height=200)
```
# What I wanted to do
If think that will be good if default screen will be current screen
```
@hook.subscribe.client_new
def func(new_window):
	if new_window.match(wmclass="pavucontrol"):
		new_window.static(screen=current_screen, x=10, y=20, width=100, height=200)
```

# What I had to do
```
@hook.subscribe.client_new
def func(new_window):
	if new_window.match(wmclass="pavucontrol"):
		qtile = new_window.qtile
		current_screen = qtile.current_screen.cmd_info()
		x = current_screen['x'] + 10
		y = current_screen['y'] + 20
		new_window.static(screen=0, x=x, y=y, width=100, height=200)

```
# Why this happens
Because in `libqtile.window` when `Static` class inits it use `self.place(x, y, width, height, 0, 0)` without participation of screen

**Comment by tych0** [comment]
Seems like a reasonable fix; can you send a patch?

**Comment by m-col** [comment]
This can be a pretty annoying big. https://github.com/qtile/qtile/pull/1865 fixes it but is just waiting for a test.

**Comment by m-col** [comment]
Fixed with #1865
